src/OSPSuite.Core/Comparison/DiffBuilder.cs

```diff
@@ -163,13 +164,17 @@ private string defaultFormatter(object input, object output)
          return output?.ToString() ?? string.Empty;
       }
 
-      private string numericFormatter<TInput>(TInput input, double value, Func<TInput, Unit> unitRetrieverFunc) where TInput : IWithDimension
+      private string numericFormatter<TInput>(TInput input, double value, Func<TInput, Unit> unitRetrieverFunc)
```

Comment:
I removed the IWIthDimension constraint so that the method can also be called with simple double values such as ValuePoint.X and Y